pages/claim/index.tsx

```diff
@@ -47,20 +49,18 @@ const Claim: NextPage = () => {
           >
             <Box w="100%" p={4}>
               <Heading size="md">
-                {hasNFT ? 'Claim your token' : "Let's get NFT first!"}
+                {hasNFT ? t('CLAIM_EXPLANATION_HEADING') : t('CLAIM_EXPLANATION_HEADING_NO_NFT')}
               </Heading>
               <Text>
                 {hasNFT
-                  ? 'You can claim your $HENKAKU.'
-                  : 'You can claim $HENKAKU if you own NFT.'}
+                  ? t('CLAIM_EXPLANATION_BODY')
+                  : t('CLAIM_EXPLANATION_BODY_NO_NFT')}
               </Text>
               {hasNFT && claimableToken != undefined ? (
                 <Text>
                   {claimableToken == BigInt(0)
-                    ? 'You currently have no points to claim'
-                    : `You can get ${ethers.utils.formatEther(
-                        claimableToken
-                      )} $HENKAKU`}
+                    ? t('CLAIM_CLAIMABLE_AMOUNT_0')
+                    : t('CLAIM_CLAIMABLE_AMOUNT_1') + ethers.utils.formatEther(claimableToken) + t('CLAIM_CLAIMABLE_AMOUNT_2')}
```

Comment:
I have no points myself, so I wasn't able to see how this displays on my browser.